azure/table_azure_storage_account.go

```diff
@@ -342,6 +348,12 @@ func tableAzureStorageAccount(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 				Transform:   transform.FromField("Account.AccountProperties.SecondaryLocation"),
 			},
+			{
+				Name:        "status_of_secondary",
+				Description: "The status indicating whether the secondary location of the storage account is available or unavailable. Only available if the SKU name is Standard_GRS or Standard_RAGRS. Possible values include: 'available', 'unavailable'.",
```

Comment:
Go SDK states `Available` and `Unavailable` with capital letters, but actual values are lowercase.

https://github.com/Azure/azure-sdk-for-go/blob/v58.3.0/services/storage/mgmt/2019-06-01/storage/models.go#L522